data-processing/entities/sentences/types.py

```diff
@@ -6,3 +6,14 @@
 @dataclass(frozen=True)
 class Sentence(SerializableEntity):
     text: str
+    cleaned_text: str
+    is_sentence: bool
```

Comment:
Why would this be false? Can the SentenceExtractor just not yield a sentence if it's not detected to be a sentence?

Reply:
If `SentenceExtractor` is completely perfect, yes. However, for debugging purposes, I kept `is_sentence` alive to see our logic of checking a line is a true sentence or not. Later, I agree that `is_sentence` should be deleted. I will make a note that this field should be deleted in the near future once we have a stable version of extractor.

Reply:
Okay sounds good. I think it's also okay to leave it in, the field should just be accompanied by a note that describes why it's there, to answer the question "wait, aren't all Sentence objects sentences?", or maybe given another name